extensions/machine-learning-services/package.json

```diff
@@ -55,7 +55,11 @@
     "commands": [
       {
         "command": "mls.command.managePackages",
-        "title": "%mls.command.managePackages%"
+        "title": "%mls.command.managePackages%",
+        "icon": {
+          "light": "./images/installPackages.svg",
+          "dark": "./images/installPackages.svg"
```

Comment:
dark light are same for now or now they always will be same?

Reply:
yes waiting for dark theme icons and colors